contents/handbook/strategy/strategy.md

```diff
@@ -5,41 +5,85 @@ showTitle: true
+
+
+### How should we prioritize between competing directions?
+
+While there is value in the items on the right, we value the items on the left more.
+
+**Breadth vs. depth of capabilities**
+
+* _Breadth_: build basic versions of every feature needed rather than a small set of sophisticated ones, so our customers can consolidate.
+
+**High quality users vs. high paying businesses**
+
+* _High quality users_: focus on acquiring more high quality users over big ticket contracts, so we can get better feedback and learn faster.
+
+**Plugin-based platform vs one-stop-shop**
+
+* _Plugin-based platform_: to move faster, look for ways to integrate with the best tools to solve customer problems over building every feature ourselves.
+
+**Self-hosted vs. Cloud**
+
+* _Self-hosted_: focus on customers who need to self-host, as there is a huge untapped market and we’re uniquely placed to win.
+
+**Reject the [“modern data stack”](https://www.analytics8.com/blog/what-is-the-modern-data-stack-and-why-should-you-be-excited-about-it/) vs. adapt to it**
+
+* _Reject_: we have a single product for ingesting, storing and analyzing data, and we will create more value for customers by making this easy to self-host than by being a small part of a cloud stack.
```

Comment:
```suggestion
* _Reject_: We enable our customers to ingest, store and analyze data on their infrastructure, we don't believe sending sensitive data to multiple cloud providers the right approach
```
@Twixes do you think this change makes the point clearer (and less contradictory)?